packages/http-server/src/__tests__/server.spec.ts

```diff
@@ -98,4 +98,17 @@ describe('server', () => {
 
     expect(response.statusCode).toBe(200);
   });
+
+  test('should return response even if there is no content defined in spec', async () => {
+    server = createServer({}, { components: {}, config: { mock: true } });
+    await server.prism.load({
+      path: relative(process.cwd(), resolve(__dirname, 'fixtures', 'no-responses.oas2.yaml')),
```

Comment:
btw I keep seeing this pop up and it can be simplified massively:

```
path: resolve(__dirname, 'fixtures', 'no-responses.oas2.yaml'),
```

This test looks great! I do not consider this an integration test, so maybe i dunno what words mean, but I like this test. A++

Reply:
Done